### ISSUE TITLE
Add a way to install/start/stop the server without cloning the repo

### ISSUE THREAD

**Issue by jellz** [jellz/minecraft-server#15]
Maybe publish the package to NPM with a little CLI attached? or use some project like [https://github.com/zeit/pkg](pkg)\* to produce a binary for admins to run that will create the necessary files (e.g. server.properties).

I'm open to other solutions, too! Feel free to reply to this issue with ideas.

\* I've never used one of these packaging tools and don't know of the caveats, I'll need to look into it more.

**Comment by lukecfairchild** [comment]
It might be nice to do a npm approach and allow it to be required into other node projects IE:
```
const server = require('.....');
const minecraft = new Server({
    port : 9001,
    more options....
});

minecraft.dostuff thats exposed.
```

**Comment by jellz** [comment]
@lukecfairchild That's a good idea and it can also be implemented but I'd like a way to use the server without touching code. Easier for server owners if it's a binary / CLI.

**Comment by lukecfairchild** [comment]
Yeah I agree having a non code based way to run it is a good plan as well.
